react-web/src/utils/utils.ts

```diff
@@ -33,4 +33,26 @@ export const formatToTitleCase = (value: string) => {
     const finalResult = result.charAt(0).toUpperCase() + result.slice(1);
     return finalResult.trim();
   }
+}
+
+export const formatTimeToReadable = (duration: number) => {
+    var milliseconds = Math.floor(duration % 1000), //Math.floor((duration % 1000) / 100),
```

Comment:
1. same for this comment. do we plan to change this computation?
2. why var?
```suggestion
    const milliseconds = Math.floor(duration % 1000),
```

Reply:
Missed to update the code, Corrected.